src/community/taskmanager/src/main/java/org/geoserver/taskmanager/web/ConfigurationPage.java

```diff
@@ -70,48 +71,48 @@
 public class ConfigurationPage extends GeoServerSecuredPage {
 
     private static final Logger LOGGER = Logging.getLogger(ConfigurationPage.class);
-    
+
```

Comment:
would you PLEASE in the future do formatting fixes in a separate commit, so I can read what you actually changed!!!!!!!!!